content/library/advanced-features/app-menu.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ☰ App menu
+slug: /library/advanced-features/app-menu
+---
+
+# ☰ App menu
+
+Streamlit provides a configurable menu within your app to access convenient tools for developers and viewers. By default, you can access developer options from the app menu when viewing an app locally or on Streamlit Community Cloud while logged into an account with administrative access. While viewing an app, click the icon in the upper-right corner to access the menu.
+
+![App menu](/images/app-menu/app-menu-developer.png)
+
+## Menu options
+
+The menu is split into two sections. The upper section contains options available to all viewers and the lower section contains options for developers. Read more about [customizing this menu](#customize-the-menu) at the end of this page.
+
+### Rerun
+
+You can manually trigger a rerun of your app by clicking "**Rerun**" from the app menu. This rerun will not reset your session. Your widget states and values stored in [`st.session_state`](/library/advanced-features/session-state) will be preserved. As a shortcut, without opening the app menu, you can rerun your app by pressing "**R**" on your keyboard (if you aren't currently focused on an input element).
+
+<div style={{ maxWidth: '30%', marginLeft: '16em' }}>
+    <Image src="/images/app-menu/app-menu-rerun-XL.png" alt="Rerun" clean />
+</div>
+
+### Settings
+
+With the "**Settings**" option, you can control the appearance of your app while it is running. If viewing the app locally, you can set how your app responds to changes in your source code. See more about development flow in [Main concepts](/library/get-started/main-concepts#development-flow). You can also force your app to appear in wide mode, even if not set within the script using [`st.set_page_config`](/library/api-reference/utilities/st.set_page_config).
+
+<div style={{ maxWidth: '30%', marginLeft: '16em' }}>
+    <Image src="/images/app-menu/app-menu-settings-XL.png" alt="Settings" clean />
+</div>
+
+#### Theme settings
+
+After clicking "**Settings**" from the app menu, you can choose between "**Light**", "**Dark**", or "**Use system setting**" for the app's base theme. Click on "**Edit active theme**" to modify the theme, color-by-color.
+
+<div style={{ maxWidth: '90%', marginLeft: '3em' }}>
+    <Image src="/images/app-menu/app-menu-settings-modal.png" alt="Settings" clean />
+</div>
+
+<br />
+
+<div style={{ maxWidth: '90%', marginLeft: '3em' }}>
+    <Image src="/images/app-menu/app-menu-settings-theme.png" alt="Theme" clean />
+</div>
+
+### Print
+
+Click "**Print**" to open your browser's print dialog. Print support is currently limited and experimental.
```

Comment:
How do we know print support is _experimental_? According to this PR https://github.com/streamlit/streamlit/pull/6180 and [spec](https://www.notion.so/snowflake-corp/Fix-printing-through-the-browser-cfcf5b720101425d8a7025c830132318), the feature is not experimental.

Reply:
Fixed on 559a704 .